### DISCUSSION TITLE
Import/export app configuration

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#8265] [General]
I remote control several PCs.
I use to configure RustDesk app (for Windows or Mac) in the same way.
It could be useful to be able to export/import configuration so I make it one time e duplicate in all apps/PCs I manage.

**Comment by Trubbleguy** [comment]
i would also like to have a way to send the export file to other client computers so they join my private server much more easily, something that can be dragged and dropped to messenger or similar. and make it available to drag and drop the file onto rustdesk window to fill in the server and key details automatically.
